drivers/project_scorecard/app/views/project_scorecard/warehouse_reports/scorecards/_report_list.haml

```diff
@@ -1,29 +1,28 @@
-- if @reports.exists?
-  %p= paginate @reports
-  .card
-    %table.table.table-striped.mb-0
-      %thead
-        %tr
-          %th Report Name
-          %th Date Range
-          %th Run On
-          %th Completed On
-          %th Author
-      %tbody
-        - @reports.each do |report|
+.table-utilities
+%p= page_entries_info @reports, entry_name: _('scorecard')
+.row
+  .col-sm-12
```

Comment:
This should be unnecessary, was it behaving oddly when not wrapped?

Reply:
Hmm nope it's not necessary, just a hasty copy-paste. I can remove.

Reply:
Thanks!

Reply:
https://github.com/greenriver/hmis-warehouse/pull/2134